.github/ISSUE_TEMPLATE/bootstrap.md

```diff
@@ -0,0 +1,69 @@
+---
+name: Bootstrap Report
+about: Issues encountered on bootstrap build system
```

Comment:
nit: Someone who's just downloaded the repo and tried to build something probably won't know that 'bootstrap' is the name of the whole build system. Maybe we could reword this a little to help clarify things

Reply:
If we call it "build system", that would cause even more confusion (with cargo) and I don't know what else we can call it.

Reply:
... Maybe Bootstrap (Build System) lol?

Reply:
Updated.